packages/react-hooks/src/inbox/use-inbox-actions.ts

```diff
@@ -60,37 +53,66 @@ interface IInboxActions {
   newMessage: (transportMessage: ICourierMessage) => void;
 }
 
+const tabsToFilters = (tabs?: ITab[], from?: number) => {
+  return tabs?.map((tab) => {
+    const filters: {
+      from?: number;
+      status?: "read" | "unread";
+    } = {
+      from,
+      status:
+        tab.filters.isRead === false
+          ? "unread"
+          : tab.filters.isRead === true
+          ? "read"
+          : undefined,
```

Comment:
what is this case?

Reply:
the filter previously was "isRead": true | false and now its "status": "read" | "unread" so making the mapping between the two kinda sucks